be/src/formats/json/binary_column.cpp

```diff
@@ -0,0 +1,91 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+#include "binary_column.h"
+
+#include "column/binary_column.h"
+#include "gutil/strings/substitute.h"
+
+namespace starrocks::vectorized {
+
+// The value must be in type simdjson::ondemand::json_type::number;
+static Status add_column_with_numeric_value(BinaryColumn* column, const TypeDescriptor& type_desc,
+                                            const std::string& name, simdjson::ondemand::value& value) {
+    std::string_view sv = value.raw_json_token();
+    column->append(Slice{sv.data(), sv.size()});
```

Comment:
Compare the length of string with the length in `type_desc` before appending, and if it's larger than the length in `type_desc`, it cannot be appended into the column.

Reply:
BTW, could you please add some unit tests to cover the above-mentioned case?

Reply:
The length check and the unit test are both added.